src/components/TablePreview/TablePreview.tsx

```diff
@@ -32,29 +40,32 @@ const TablePreview: FC<Props> = ({}) => {
 
     return (
         <Flex direction="column" height="100%" alignItems="center" marginBottom="size-1200">
-            <table ref={tableRef}>
-                <tbody>
-                    <tr>
-                        <td rowSpan={3} align="center">
-                            <img src={imgUrl} height="60"></img>
-                        </td>
-                        <td style={name.style}>{name.textValue}</td>
-                        <td style={spacerStyle}></td>
-                        <td style={phone.style}>{phone.textValue}</td>
-                    </tr>
-                    <tr>
-                        <td style={title.style}>{title.textValue}</td>
-                        <td style={spacerStyle}></td>
-                        <td style={email.style}>{email.textValue} </td>
-                    </tr>
-                    <tr>
-                        <td style={company.style}>{company.textValue}</td>
-                        <td style={spacerStyle}></td>
-                        <td style={linkedin.style}>{linkedin.textValue}</td>
-                    </tr>
-                </tbody>
-            </table>
-
+            {hasData ? (
+                <View borderWidth="thin" borderColor="dark" borderRadius="medium" padding="size-250">
+                    <table ref={tableRef}>
+                        <tbody>
+                            <tr>
+                                <td rowSpan={3} align="center">
+                                    <img src={imgUrl} height="60"></img>
+                                </td>
+                                <td style={name.style}>{name.textValue}</td>
+                                <td style={spacerStyle}></td>
+                                <td style={phone.style}>{phone.textValue}</td>
+                            </tr>
+                            <tr>
+                                <td style={title.style}>{title.textValue}</td>
+                                <td style={spacerStyle}></td>
+                                <td style={email.style}>{email.textValue} </td>
+                            </tr>
+                            <tr>
+                                <td style={company.style}>{company.textValue}</td>
+                                <td style={spacerStyle}></td>
+                                <td style={linkedin.style}>{linkedin.textValue}</td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </View>
+            ) : null}
             <ToastContainer />
```

Comment:
🤔 Looks like this `ToastContainer` slipped in, we shouldn't need one so deep in the codebase, we should only need one closer to the top of the application

Reply:
Roger on that, lifting the ToastContainer to a upper level of the application! maybe at the level of providers? 🤔

Reply:
There's already one up there! 

https://github.com/evargast/signature-generator/blob/main/src/components/App/App.tsx#L18